test/index.test.js

```diff
@@ -1,24 +1,29 @@
 import { expect } from 'chai';
-import { srcSet, getWidths, constructURL, imgSizes } from '../src/Utils';
+import { srcSet, constructURL, imgSizes } from '../src/Utils';
 
 describe('Utils tests', () => {
   const timeout = 3000;
 
-  test('should get the correct widths', async () => {
-    const widths = getWidths(2440, 4880);
-    expect(widths.length).to.equal(2);
-    expect(widths[0]).to.equal(1220);
-  }, timeout);
-
   test('should construct correct srcSet', async () => {
+    const presetImageSizes = [
+      640,
+      768,
+      1024,
+      1366,
+      1600,
+      1920,
+    ]
```

Comment:
i think we can export `presetImageSizes` from `src/index.js` so we can re use it here

Reply:
thanks! fixed in https://github.com/hiburrow/graphcms-image/pull/3/commits/1f608c94e498deec278e0bf355ba85c8a93a884f